Hermes/appscale/hermes/__init__.py

```diff
@@ -12,30 +10,30 @@
 import socket
 import sys
 import tarfile
+import urllib
+
 import tornado.escape
 import tornado.httpclient
 import tornado.web
-import urllib
-
 from appscale.common import appscale_info
 from appscale.common import appscale_utils
+from appscale.common.constants import LOG_FORMAT
 from appscale.common.ua_client import UAClient
 from appscale.common.ua_client import UAException
-from handlers import MainHandler
-from handlers import TaskHandler
-from helper import JSONTags
-
+from appscale.common.unpackaged import APPSCALE_PYTHON_APPSERVER
 from tornado.ioloop import IOLoop
 from tornado.ioloop import PeriodicCallback
-from tornado.options import define
 from tornado.options import options
-from tornado.options import parse_command_line
 
-sys.path.append(os.path.join(os.path.dirname(__file__), '../AppServer'))
+from appscale.hermes import constants
```

Comment:
Yeah that makes sense @jeanleonov and I agree with having some standards in place for ordering of imports. My comment was basically towards what we have been following so far and we can change that to this new approach by making a conscious effort to fix the ordering in whichever class we work on.